jobs/copilot/spec

```diff
@@ -40,6 +41,10 @@ properties:
   listen_port_for_mcp:
     description: "The port that Pilot MCP client will connect to, to retrieve route info"
     default: 9009
+  dns_address_for_mcp:
```

Comment:
I find it confusing that the address that Pilot uses to connect to copilot via MCP is under the copilot job. I would expect this to be under the pilot job. I'm guessing we're leveraging BOSH DNS, so copilot registers this DNS name?  I see below that we're configuring Pilot to pull the address it uses from the copilot property. 

However, if the user of this release isn't going to use copilot and wants to configure Pilot to use a different MCP server, this UX is pretty weird. Maybe it would make more sense if Pilot had its own properties for port and address, and we used BOSH links to pull them from copilot by default but they could be overridden? (I'm not entirely sure links work that way)

Reply:
I think the idea is that a different mcp server would provide an `mcp_conn` link that has both the `listen_port_for_mcp` and `dns_address_for_mcp` properties set i.e you wouldn't change `copilot` link property to use a different server, you would provide your own.

Reply:
@christianang Exactly. We would provide a link in a completely separate Bosh release. Copilot is neither configured nor used at all.
@shalako Does it make more sense like this or do you still think we should go for a simple property in the pilot job?